system/ee/ExpressionEngine/app.setup.php

```diff
@@ -263,6 +264,18 @@
             );
         },
 
+        'Resource' => function () {
+            return new Resource\Request();
+        },
+
+        'Resource/Javascript' => function () {
+            return new Resource\Javascript();
```

Comment:
@robsonsobral This is where the `Javascript` class is instantiated and the "s" is lowercase. PHP class names are technically case insensitive, but they really should be consistent to avoid any confusion.